src/main/routes/receiver.ts

```diff
@@ -35,8 +35,9 @@ const eligibilityStore = new CookieEligibilityStore()
 
 async function getOAuthAccessToken (req: express.Request, receiver: RoutablePath): Promise<string> {
   if (req.query.state !== OAuthHelper.getStateCookie(req)) {
-    throw new Error('Invalid state')
+    logger.warn('State cookie mismatch! Request: "' + req.query.state + '", Cookie: "' + OAuthHelper.getStateCookie(req) + '"')
```

Comment:
Application Insights collects data from logger, so this is logged on Azure, I have checked this. Story mentions to log App Insights 'event' so I am not sure this solution is 100% fine.